draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -437,13 +442,13 @@ The Issuer creates the following Disclosures:
 
 {{examples/simple/disclosures.md}}
 
-The SD-JWT is then signed by the Issuer to create a JWT like the following:
+The payload is then signed by the Issuer to create a JWT like the following:
 
-<{{examples/simple/sd_jwt_serialized.txt}}
+<{{examples/simple/sd_jwt_jws_part.txt}}
 
 ## Decoy Digests {#decoy_digests}
 
-An Issuer MAY add additional digests to the SD-JWT that are not associated with
+An Issuer MAY add additional digests to the SD-JWT payload that are not associated with
```

Comment:
```suggestion
An Issuer MAY add additional digests to the Issuer-signed JWT payload that are not associated with
```

Reply:
IMHO "SD-JWT payload" is okay here. "Issuer-signed JWT" is okay too but I'd like to not use that too much and so only when the distinction is needed.

Reply:
ok..